test_mvp.ipynb

```diff
@@ -2,56 +2,157 @@
  "cells": [
```

Comment:
<p>This plot looks very similar to the previous, I expected a the peak at 4.6 not after 5. Are you sure the plot is up to date with the sampled parameters?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sequgen/notebooks/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>We found out that placement refers to the start of the peak (left part of triangle)</p>

Reply:
thanks for the explaination